x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TransformStatsTests.java

```diff
@@ -89,4 +89,32 @@ public void testBwcWith73() throws IOException {
             }
         }
     }
+
+    public void testBwcWith76() throws IOException {
```

Comment:
We also need to be BWC with de-serializing an old stats document via XContent.

Reply:
with true lenient parsing this is not necessary anymore